ezResx/Solution/SolutionReader.cs

```diff
@@ -110,7 +111,7 @@ public List<ResourceItem> GetSolutionResources(string solutionPath)
                         var resourceItem = resourceList.FirstOrDefault(x => Equals(x.Key, resourceKey));
                         if (resourceItem == null)
                         {
-                            throw new Exception($"No invariant culture resource found for {resourceKey.File} {resourceKey.Name}");
+                            throw new Exception ($"No invariant culture resource found for {resourceKey.File} {resourceKey.Name}");
```

Comment:
File contains 2 changes. None of them is functional + this change is a bit against code style conventions. Should be undone.